Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -1,268 +1,33 @@
+import Foundation
```

Comment:
All the contents in this file are new. Probably best/easiest to review the whole thing by clicking `View file`.